### PR TITLE
Resolved #983, #1279, #1293, #1313, #1314 & updated Polish translations

### LINKED ISSUE #983: Exception on undoing too much in text field.
Brief Description: When you press ctrl-z to undo, in a text field, while there's nothing to undo, you get an exception. It doesn't seem to break anything, but it should probably be fixed at some point.

Operating System (OS and version): Windows 7
OpenBazaar version: 1.0.7

Reproducible: Always

Steps to reproduce:
1. Click any text field
2. Press ctrl-z (or whatever undo is on your system) until there's nothing to undo, and then once more.
3. You get an exception. You will get another error when you close the client.

### LINKED ISSUE #1279: possible addition of F5 reload page feature
is it possible to enable F5 to refresh pages?

### PR DESCRIPTION
**Pull request by Libter**

### CODE REVIEW

**Comment by jjeffryes** [comment]
In the future, it would be better to make translation PRs separate from bugfix PRs, so it's easier for us to review and recommend changes.

File: css/obBase.css [hunk 1]
```diff
@@ -1176,7 +1176,7 @@ label.fieldItem {
   z-index: 11;
   top: 50px;
   height: 100%;
-  width: calc(100% - 30px);
+  width: 100%;
```
Comment (jjeffryes) [inline css/obBase.css:1179]: The image overlay should not overlap the sidebar.

Comment (Libter) [inline css/obBase.css:1179]: @jjeffryes in this case #1320 is better solution.

File: js/main.js [hunk 2]
```diff
@@ -185,43 +185,54 @@ $('body').on('keypress', 'input', function(event) {
...
+  var char = String.fromCharCode(e.which).toLowerCase();
+  var ctrl = (e.ctrlKey || e.metaKey) && !e.altKey; //test for alt key to prevent international keyboard issues
+    
+  if (event.keyCode == 116) { //on F5 press
+    location.reload();
+  }
+  
+  if (ctrl && char == 'z') { //run undo programmatically to avoid crash
+    e.preventDefault();
+    document.execCommand('undo');
```
Comment (jjeffryes) [inline js/main.js:197]: This is a smart way to fix the undo bug. Good work.

**Pull request merged**